src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Register</title>
+ <link rel="stylesheet" href="/css/main.css">
+ <style>
+   label {
+     display: inline-block;
+     width: 100px;
+   }
+ </style>
+</head>
+<body>
+
+ <nav>
+   <a id="navTitle" href="/">CodeU Chat App</a>
+   <a href="/conversations">Conversations</a>
+   <% if(request.getSession().getAttribute("user") != null){ %>
+     <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+   <% } else{ %>
+     <a href="/login">Login</a>
+   <% } %>
+ </nav>
+
+ <div id="container">
+   <h1>Register</h1>
+
+   <form action="/register" method="POST">
+     <label for="username">Username: </label>
+     <input type="text" name="username" id="username">
+     <br/>
+     <label for="password">Password: </label>
+     <input type="password" name="password" id="password">
+     <br/><br/>
+     <button type="submit">Submit</button>
+   </form>
+ </div>
+</body>
+</html>
```

Comment:
Git added a symbol saying that there is no new line at the end of the file.  I don't know if this impacts the code at all, I know in C it does sometimes after comments  but I assume it doesn't here, just wanted to point it out.

Reply:
I looked at some things on stack overflow.  Most people say it shouldn't affect anything unless it's c/c++ header files but a couple of people were very passionate that it's a terrible habit.